### ISSUE TITLE
why there are 3 training step?

### ISSUE THREAD

**Issue by fire717** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#195]
Nice work!
In your training guide, there are 3 different step:

1. --from-mobilenet
2. --weights-only
3.  --num-refinement-stages 3

there are two question:
1. from 2 to 3, i can understand it that you need to train the front refinement stage to get stable output and then to train the rest stage, is it right? Then did you have tried that if train 3 stage directly, whats the AP different from training separately?
2. what's the different between 1 and 2? It seems that 2  just load the weight from 1, and not load optimizer,scheduler,etc. Means that just training from iter 1 with trained weight? If is so, Why is it necessary?

Looking forword to your reply, thx!

**Comment by Daniil-Osokin** [comment]
Hi! This is kind of legacy, see #178. You can train with all 6 stages from the beginning. The only trick is to train 2<sup>nd</sup> time using weights from previous training. It usually helps to improve accuracy.

**Comment by fire717** [comment]
help a lot. thx!

**Comment by Daniil-Osokin** [comment]
Ok, closing.
